src/Sql/Sql.Test/ScenarioTests/LedgerDigestUploadTests.ps1

```diff
@@ -0,0 +1,194 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests getting default ledger digest upload configuration
+#>
+function Test-GetDefaultLedgerDigestUpload
+{
+	# Setup
+	$testSuffix = getAssetName
+	$params = Get-LedgerTestEnvironmentParameters $testSuffix
+	Create-LedgerTestEnvironment $params
+
+	$databaseResourceId = "/subscriptions/" + $params.subscriptionId + "/resourceGroups/" + $params.rgname + "/providers/Microsoft.Sql/servers/" + $params.serverName + "/databases/" + $params.databaseName 
+	$databaseObject = Get-AzSqlDatabase -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName
+
+	try 
+	{
+		# Test
+		$ledgerDigestUpload = Get-AzSqlDatabaseLedgerDigestUpload -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName
+	
+		# Assert
+		Assert-AreEqual $ledgerDigestUpload.State "Disabled"
```

Comment:
So we only test Get for Disabled? Are the other scenarios tested as part of the Set tests?